src/blog/_posts/2018-11-06-opera-57.md

```diff
@@ -0,0 +1,151 @@
+
+    window.postMessage('my message', { targetOrigin: 'https://origin.example.com' });
+
+## Media and Real-time Communications
+
+* [RTCPeerConnection.prototype.getConfiguration](https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-getconfiguration)
+  has been added to allow accessing the last configuration set by
+  [RTCPeerConnection.prototype.setConfiguration](https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-setconfiguration).
+* The `Opus` codec is now supported in mp4 (ISO-BMFF) containers with Media Source Extensions (MSE).
+* The new SourceBuffer.changeType() method allows for improved cross-codec or cross-bytestream transitions
+  within a SourceBuffer. It would for example allow changing the type of media bytes append to a media stream.
+
+## Network
+
+### Intervention Reports
+
+An "intervention" happens when the user agent (the browser) willfully violates
+the request of a web application in order to improve performance, security or
+eliminate behavior that is considered annoying to a user. These interventions
+can now be reported either by using the
+[`Report-To` HTTP Response header](https://w3c.github.io/reporting/#header) or the
+[`ReportingObserver` API](https://w3c.github.io/reporting/#observers).
+
+### TLS 1.3
+
+[TLS 1.3](https://tools.ietf.org/html/rfc8446) is a new version of the TLS
+protocol that sports a simpler, less error-prone design which improves both
+efficiency and security. This is achieved by, among other things, reducing the
+number of round-trips needed to setup a connection, and remove a number of
+insecure legacy options.
+
+## Layout / CSS
+
+* The `<rp>` element is now `display: none` by default even outside a `<ruby>` element.
+* The behavior of percentage row tracks and gutters in grid containers with
+indefinite heights has been updated, and now behaves the same as (grid) columns.
+
+##  Miscellanea
+
+### `Symbol.prototype.description`
+
+`Symbol.prototype.description` provides a more straight-forward way to access
+the description of a `Symbol` rather than the previously more indirect access
+via `Symbol.prototype.toString`.
+
+### WebUSB on Dedicated Workers
+
+The WebUSB feature can now be accessed from within a dedicated worker. This for
+example allows communication with an USB device to be offloaded from the main
+thread - which can improve performance.
+
+## Removed features
+
+### AppCache in non-secure contexts
+
+The Application Cache feature, after having been deprecated since Opera 54
+(Chromium 67), is no longer exposed in non-secure contexts.
+
+### Anonymous getter on HTMLFrameSetElement
+
+It was previously possible to access `<frame>` elements by name via their
+parent `<frameset>`. This was a non-standard extension, which has now been
+removed.
+
+## Deprecated features
+
+### Custom Elements "V0" and Shadow DOM "V0"
+
+The "V0" versions of the Custom Elements and Shadow DOM APIs has been
+deprecated, and is expected to be removed in Opera 60 (Chromium 73.) Please
```

Comment:
Period outside the parenthesis. 
https://english.stackexchange.com/questions/6632/where-does-the-period-go-when-using-parentheses

Reply:
Fixed.